### PR TITLE
Server db connection

### PR DESCRIPTION
**Pull request by bkwon94**
Starting code for my server and database setup.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+app.use(bodyParser.urlencoded( { extended: true } ))
+app.use(bodyParser.json())
+app.use(cors());
+
+app.listen(PORT, () => {
+  console.log(`Listening on port ${PORT}`);
+})
+
+// get request example
+app.get('/movies', (req, res) => {
```
Comment (brockross) [inline server/index.js:18]: have you verified using Postman/curl that this route returns the db data as expected?

File: db/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/fec', { useNewUrlParser: true});
```
Comment (Danielxburns) [inline db/index.js:2]: is useNewUrlParser necessary for this project?

File: server/index.js (hunk 1 above)
Comment (bkwon94) [reply server/index.js:18]: Brock, 
Yes I have tested with Postman and indeed I receive the db data as expected. Great question.

File: db/index.js (hunk 2 above)
Comment (bkwon94) [reply db/index.js:2]: Daniel,
useNewUrlParser seems to be recommended to avoid any deprecation warning.

**Review by Danielxburns — APPROVED** [review]

**Pull request merged**